example.php

```diff
@@ -1,7 +1,8 @@
 <?php
 
 $foo = NULL;
-// is_null is a function that does stuff 
+// This is an entirely uninitiated interactive rewrite
 is_null($foo);
 
-?>
\ No newline at end of file
+// Hello world
```

Comment:
```diff
- // The original line
+ // Some {{ .commentVar }}
```

Change `commentVar` by commenting: `@CodeLingoBot set commentVar "your comment"`

Reply:
@CodeLingoBot set commentVar "your comment"

Reply:
@CodelingoBotTester set commentVar "your comment"

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite set commentVar "your comment"

Reply:
error handling failed for:POST https://api.github.com/repos/Daanikus/interactive-rewrite-test/pulls/22/comments: 422 Validation Failed [{Resource:PullRequestReviewComment Field:pull_request_review_id Code:custom Message:pull_request_review_id must be pending}]: POST https://api.github.com/repos/Daanikus/interactive-rewrite-test/pulls/22/comments: 422 Validation Failed [{Resource:PullRequestReviewComment Field:pull_request_review_id Code:custom Message:pull_request_review_id must be pending}]

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>